core/overlays/middletier-stage/configmaps.yaml

```diff
@@ -25,8 +26,8 @@ data:
       archiveLogs: False
 
       s3:
-        bucket: "DH-DEV-DATA"
-        keyPrefix: "data/thoth/thoth-middletier-stage/argo/artifacts/"
+        bucket: "thoth"
+        keyPrefix: "data/ocp-stage/argo/artifacts/"
```

Comment:
`ocp-prod-stage`?

Reply:
if we keep ocp-stage, it would have same consistency as 
<cluster-name>-<env-type> 
Ex: ocp-stage, ocp-test, moc-prod, psi-stage, psi-test 
earlier we had prefix of thoth , like thoth-psi-test , we can have it that way too

Reply:
Sounds good to me :) Thanks!